This patch is needed in order to prevent automake to create an absolute path for srcdir in our build environment.

--- maxima-5.50.0/tests/Makefile.am.orig
+++ maxima-5.50.0/tests/Makefile.am
@@ -88,6 +88,6 @@
 endif
 
 genericdir = $(insttestsdir)
-genericdirDATA = $(wildcard $(addprefix $(srcdir)/, README.how-to mcond_display_elseif.out *.mac *.lisp *.wxm wester_problems/*))
+genericdirDATA = $(wildcard README.how-to mcond_display_elseif.out *.mac *.lisp *.wxm wester_problems/*)
 
 EXTRA_DIST = $(genericdirDATA) test.sh.in
